.github/workflows/docker-build-and-test.yml

```diff
@@ -302,6 +302,95 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Get coreboot version
+        id: version
+        if: startsWith(matrix.dockerfile, 'coreboot')
+        run: yq -r '.services.["${{ matrix.dockerfile }}"].build.args[] | select(test("COREBOOT_VERSION=.*"))' docker/compose.yaml >> "${GITHUB_OUTPUT}"
+      - name: Clone coreboot
+        if: startsWith(matrix.dockerfile, 'coreboot')
+        run: |
+          git clone --depth 1 "https://review.coreboot.org/coreboot.git" -b "${{ steps.version.outputs.COREBOOT_VERSION }}"
+      - name: Get coreboot commit hash
+        id: coreboot-hash
+        if: startsWith(matrix.dockerfile, 'coreboot')
+        run: |
+          cd coreboot
+          COREBOOT_HASH="$( git rev-parse --short HEAD )"
+          echo "${COREBOOT_HASH}"
```

Comment:
What is the purpose of outputting the content of the variable here? Just for debugging? And in the next step you out `CACHE_KEY=${CACHE_KEY}` aka `KEY=VALUE` and here just `VALUE`. Was that done on purpose?

Reply:
Just for debugging.